.gitignore

```diff
@@ -1 +1,2 @@
-credentials.sql
\ No newline at end of file
+credentials.sql
+previewConfig.json
```

Comment:
This file is used by preview js which is a VS Extenshion. This is not needed in Branch and is only needed locally

Reply:
You can add a gitignore global file and add 
`.vscode`
This way you don't need to add these local setup things to the `gitignore`. Otherwise, if everyone added their local configs to the `gitignore` file, we could end up with a giant one :sweat_smile: 

read more about this here https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Thanks @mivasconcelos 

I did have the sam question but didn't know about this. Should have researched.